validator/sawtooth_validator/journal/responder.py

```diff
@@ -130,7 +129,7 @@ def handle(self, connection_id, message_content):
 
             block_response = network_pb2.GossipBlockResponse(
                 content=block.get_block().SerializeToString(),
-                node_id=node_id)
+                )
```

Comment:
This might be a tad pedantic, but would suggest removing the trailing comma and moving the close paren either to the previous line or the same tab spacing as the opening line

Reply:
updated